src/typescript/frontend/src/components/footer/index.tsx

```diff
@@ -21,7 +21,7 @@ const Footer: React.FC = () => {
         <StyledContainer>
           <StyledSocialWrapper>
             <StyledClickItem>
-              <LogoIcon width="170px" cursor="pointer" versionBadge={true} />
+              <LogoIcon width="170px" cursor="pointer" versionBadge={true && !isMobile} />
```

Comment:
You could also just change the flex style (from row to col) for mobile so that they're stacked on top of each other and don't add horizontal space

Reply:
I think that was the issue this PR was supposed to fix in the first place, the fact that they would stack up on mobile.

Reply:
- [ ] If resizing to a small screen, put the alpha badge below, but enforce no line breaking inside the badge